docs/introduction.rst

```diff
@@ -304,4 +304,60 @@ Cons
 
 Saliency Algorithms
 --------------------------------
-*Discuss the provided xaitk-saliency algorithms in terms of the above.*
+.. Discuss the provided xaitk-saliency algorithms in terms of the above.
+
+The xaitk-saliency API consists of a number of object-oriented interfaces for saliency heatmap generation, which
+focus on black-box visual saliency. Three types of implementations of these interfaces are provided: Image
+Perturbation, Heatmap Generation, and End-to-End Saliency Generation.
+
+
+Image Perturbation
+^^^^^^^^^^^^^^^^^^^
+These implementations are occlusion-based saliency algorithms that perform perturbations on image pixels.
+
+* RISEGrid - Transforms an input reference image into a number of mask matrices indicating the perturbed regions and
+  generates an importance map indicating how salient each pixel is for the model's prediction.
+  See the "DRISE" Jupyter Notebook example.
```

Comment:
Updating based on the fact that this implementation doesn't involve saliency computation.

```suggestion
* RISEGrid - Transforms an input reference image into a number of grid-based mask matrices indicating the perturbed regions. See the "DRISE" Jupyter Notebook example.
```

Reply:
done